.github/release.yml

```diff
@@ -0,0 +1,21 @@
+changelog:
+  exclude:
+    labels:
+      - no-changelog
+    authors:
+      - dependabot
+      - pre-commit-ci
+  categories:
+    - title: Breaking Changes 🛠
+      labels:
+        - breaking-change
+    - title: Enhancements 🎉
+      labels:
+        - enhancement
+        - "*"
+    - title: Security hardening 🔒
```

Comment:
Wondering if `hardening` here should be `Hardening` instead, just to follow the other two titles that have an upper-case on the second word.